tag/cli/locuspocus.py

```diff
@@ -58,4 +58,5 @@ def main(args):
         minperc=args.min_perc
     )
     writer = tag.writer.GFF3Writer(locusstream, args.out)
+    writer.complex_separators = False
```

Comment:
Here, we want separators to mark boundaries between loci, not between features within a locus.